src/pages/AddItem.js

```diff
@@ -40,7 +26,7 @@ const AddItem = () => {
   const handleSubmit = event => {
     event.preventDefault();
     setError(isDuplicate(name));
-    addItem(name, nextExpectedPurchase);
+    addItem(name, nextExpectedPurchase, token);
```

Comment:
This is smart! Passing in the token to `addItem` means that's one less responsibility for that function. Love this!